### PR TITLE
Fixing talk data

### PR DESCRIPTION
**Pull request by bethaitman**
Fixing up little issues I find while I'm doing the index.

### CODE REVIEW

**Comment by bethaitman** [comment]
One of those fixes worked and one of them didn't. Just trying to work out what's wrong.

File: docs/_data/2017.na.speakers.yaml [hunk 1]
```diff
@@ -1090,6 +1090,7 @@
     title: Intelligent Documents and the Verifiability Crisis in Science, Tech Writing,
         and Life
     year: 2017
+    youtubeId: kOzQPpc-KDo
```
Comment (plaindocs) [inline docs/_data/2017.na.speakers.yaml:1093]: wonder if you need to quote the id because of the hyphen?

Comment (bethaitman) [reply docs/_data/2017.na.speakers.yaml:1093]: No, that's working fine. It's the other one that's broken!

File: docs/_data/2017.eu.speakers.yaml [hunk 2]
```diff
@@ -509,7 +509,7 @@
     title: 'Tech Writers Without Borders: Making the world a better place, one (numbered) step at a time'
     slides: https://speakerdeck.com/sculshaw/making-the-world-a-better-place-one-numbered-step-at-a-time
     event: Write the Docs EU 2017
-    path: conf/eu/2017/videos/
+    path: conf/eu/2017/videos/tech-writers-without-borders-making-the-world-a-better-place-one-numbered-step-at-a-time-stuart-culshaw
```
Comment (bethaitman) [inline docs/_data/2017.eu.speakers.yaml:512]: For some reason this isn't working

Comment (bethaitman) [reply docs/_data/2017.eu.speakers.yaml:512]: His link seems still to be broken https://deploy-preview-828--dentist-canary-64055.netlify.com/videos/eu/2017/

File: docs/_data/2017.eu.speakers.yaml [hunk 3]
```diff
@@ -509,7 +509,7 @@
...
     series: Write the Docs EU
     series_slug: eu
     slug: ''
```
Comment (plaindocs) [inline docs/_data/2017.eu.speakers.yaml]: @bethaitman  I reckon this is the problem, not the path. Or as well as the path.

Comment (bethaitman) [reply docs/_data/2017.eu.speakers.yaml]: Genius. I didn't spot that

**Review by plaindocs — APPROVED** [review]

**Pull request merged**